sig/datadog/appsec/processor/context.rbs

```diff
@@ -0,0 +1,27 @@
+module Datadog
+  module AppSec
+    class Processor
+      class Context
+        type event = untyped
```

Comment:
I think `event` is a ::Hash[Symbol, untyped] according to where it's used (e.g.: https://github.com/DataDog/dd-trace-rb/blob/master/lib/datadog/appsec/contrib/rack/gateway/watcher.rb#L47), I'm not sure it is more useful than declaring it as untyped though
```suggestion
        type event = ::Hash[Symbol, untyped]
```

Reply:
Maybe, I'm not sure that I fully understand the code to change it now, we have events from the `WAF` module too and so far they are always empty.

I think we can definitely change it as soon as we 100% sure.